Makefile

```
@@ -37,7 +37,7 @@ unittest:
 	@printf '\n\n*****************\n'
 	@printf '$(color)Running unittest$(off)\n'
 	@printf '*****************\n'
-	$(PYTHON) -m unittest tests/test_flexmock.py
+	PYTHONPATH=$(shell pwd) $(PYTHON) tests/test_unittest.py
```

What's the difference with this ?
```suggestion
	PYTHONPATH=$PWD $(PYTHON) tests/test_unittest.py
```

---

Most of the times it is probably the same, but there is no guarantee that `$PWD` environment variable exists so running `pwd` command with `shell pwd` is a bit safer.